authors/apps/authentication/templates/email_confirm.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+.card {
+ box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+ max-width: 600px;
+ margin: auto;
+ text-align: center;
+ font-family: arial;
+}
+
+.title {
+ color: grey;
+ font-size: 13px;
+ text-align: left;
+}
+
+button {
+ border: none;
+ outline: 0;
+ display: inline-block;
+ padding: 8px;
+ color: #000;;
+ background-color: #000;
+ text-align: center;
+ cursor: pointer;
+ width: 100%;
+ font-size: 18px;
+}
+
+a {
+ text-decoration: none;
+ font-size: 13px;
+ color: black;
+}
+
+button:hover, a:hover {
+ opacity: 0.7;
+}
+</style>
+</head>
+<body>
+
+<div class="card">
+    <p class="title">
+    Hi {{username}},<br></p>
+    <p class="title"> Welcome to <a href="http://ah-orcas-staging.herokuapp.com/">
```

Comment:
Can the URL be dynamic

Reply:
we placed it that way because the Url to the staging is static.. what do you suggest?

Reply:
we could have an environment variable in place, something like:
https://ah-orcas-[environment].herokuapp.com

Reply:
mmmh, okay. let us try that out